### ISSUE TITLE
[question]: Multiple commands produce [...]

### ISSUE THREAD

**Issue by rum-n** [OneSignal/react-native-onesignal#1485]
### How can we help?

I've been getting the error:

Multiple commands produce '[path]/Build/Products/Debug-iphonesimulator/project.app/Frameworks/OneSignalOutcomes.framework'

and 

Multiple commands produce '[path]/Build/Products/Debug-iphonesimulator/project.app/Frameworks/OneSignalCore.framework'

Does anyone know how to solve this?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by NikitaNS** [comment]
Have the same problem. RN: 0.71.1, React: 18.2.0, OneSignal: 4.5.0

**Comment by PatrickBokhove** [comment]
I was running in to the same issue, and here is how i solved it:

In the Build Phases tab of your main target is a section called Embed Frameworks. This section includes `OneSignalCore.xcframework` and `OneSignalOutcomes.xcframework`. However, these files probably also get embedded in the Embed Pods Frameworks section. After deleting the files from the Embed Frameworks section, my app built just fine again. In the General tab of your main target, there's a section called Frameworks, Libraries, and Embedded Content. After removing the frameworks from Embed Frameworks these should now be labeled as Do Not Embed.

<img width="1339" alt="Scherm­afbeelding 2023-02-15 om 13 26 02" src="https://user-images.githubusercontent.com/7600895/219060998-bd867515-b005-43dc-8980-2fb85d5ced90.png">

<img width="1204" alt="Scherm­afbeelding 2023-02-15 om 15 49 44" src="https://user-images.githubusercontent.com/7600895/219061560-d7668b4c-903a-4574-8727-7ad611e7890f.png">

**Comment by BogdanRad** [comment]
Hello, I have same error only for M1.  On intel it's working perfectly... The solution from @PatrickBokhove it's working. Thank you!

**Comment by hoseinfo** [comment]
same issue here...

**Comment by indapublic** [comment]
Lock on 4.5.0 version works for me (after that you need to force reinstall package and pods)

```
"react-native-onesignal": "4.5.0"
```

**Comment by TiagoAndreAlmeida** [comment]
I still getting this erro... trying everything but no success....

**Comment by HamzaBakaran** [comment]
I faced the same issue today, tried all the steps, and it is not working. Has anyone found (another) solution?

**Comment by phuocantd** [comment]
same here

**Comment by quachdev** [comment]
Same here as well - running into this issue now when trying to install `react native firebase`. I assume it's something to do with the usage of `use_frameworks!`. 

@PatrickBokhove comment doesn't look to be relevant for OneSignal `v4.5.3`.  The embed frameworks section is empty now.

**Comment by PrimulaX** [comment]
Has anyone resolved this issue? Trying to use Firebase with `use_frameworks!` with no success

**Comment by quachdev** [comment]
> Has anyone resolved this issue? Trying to use Firebase with `use_frameworks!` with no success

This helped me for Firebase: https://stackoverflow.com/questions/72289521/swift-pods-cannot-yet-be-integrated-as-static-libraries-firebasecoreinternal-lib/74432488#74432488

TL;DR can just set specific Firebase dependencies rather than everything

So my podfile looks something to this: 
```
  pod 'Firebase', :modular_headers => true
  pod 'FirebaseCoreInternal', :modular_headers => true
  pod 'GoogleUtilities', :modular_headers => true
  pod 'FirebaseCore', :modular_headers => true
  $RNFirebaseAsStaticFramework = true
```

**Comment by PrimulaX** [comment]
@quachdev My Podfile looks the same but I still have a problem

**Comment by Jeadonara** [comment]
I guess it is the best solution as mentioned here https://github.com/OneSignal/OneSignal-Flutter-SDK/issues/551#issuecomment-2363339243

`target 'OneSignalNotificationServiceExtension' do
  use_frameworks! :linkage => :static
  pod 'OneSignalXCFramework', '>= 3.0', '< 4.0'
end`

**Comment by sherwinski** [comment]
Hello, apologies for the delay here. Closing this as it looks like a couple of solutions have been presented. Please feel free to ping me if it still remains an issue.
